### PR TITLE
A3: Press Release

### PR DESCRIPTION
**Pull request by Brandonz07**
i done html and css

### CODE REVIEW

File: src/br/html/A3-Press-Release.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+          <a href="#"><i class="fa-brands fa-facebook"></i></a>
+          <a href="#"><i class="fa-brands fa-x-twitter"></i></a>
+          <a href="#"><i class="fa-solid fa-envelope"></i></a>
+          <a href="#"><i class="fa-regular fa-clipboard"></i></a>
+        </div>
+      </div>
+
+      <div class="imgs">
+        <img src="./img/img1.jpg" alt="" />
+        <p class="txt-center">
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release.html:46]: The correct tag here is a figcapion tag, Note that in that case you don't need a property '.imgs p'
please refactor it.

File: src/br/html/A3-Press-Release.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <main>
+      <div class="txt-center">
+        <h1>
+          Apple’s global phenomenon “Ted Lasso” joins ranks of the most
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release.html:22]: The font used in this document is very different from the model.
Find a suitable source for this document.

File: src/br/html/A3-Press-Release.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+          <li>Outstanding Short Form Comedy, Drama or Variety Series</li>
+        </ul>
+
+        <h3>“Schmigadoon!”</h3>
+
+        <ul>
+          <li>Outstanding Original Music and Lyrics</li>
+        </ul>
+
+        <h2>“Carpool Karaoke: The Series”</h2>
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release.html:163]: Note that the heading of this level have a slightly different margins or padding
please refactor it.

File: src/br/html/A3-Press-Release.html [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+      </div>
+
+      <div class="imgs">
+        <img src="./img/img1.jpg" alt="" />
+        <p class="txt-center">
+          “Ted Lasso” took home four wins at the 74th Primetime Emmy Awards,
+          including back-to-back wins for Outstanding Comedy Series for both its
+          first and second seasons.
+        </p>
+      </div>
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release.html:51]: An arrow element is missing here.
please include it.

File: src/br/html/A3-Press-Release.html (hunk 1 above)
Comment (Brandonz07) [reply src/br/html/A3-Press-Release.html]: now use figcaption

File: src/br/html/A3-Press-Release.html (hunk 2 above)
Comment (Brandonz07) [reply src/br/html/A3-Press-Release.html:22]: change the font

File: src/br/html/A3-Press-Release.html (hunk 3 above)
Comment (Brandonz07) [reply src/br/html/A3-Press-Release.html]: add padding

Comment (Brandonz07) [reply src/br/html/A3-Press-Release.html]: add margin

File: src/br/html/A3-Press-Release.html (hunk 4 above)
Comment (Brandonz07) [reply src/br/html/A3-Press-Release.html:55]: I included the arrow element

File: src/br/html/A3-Press-Release/A3-Press-Release.html [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+          <a href="#"><i class="fa-brands fa-facebook"></i></a>
+          <a href="#"><i class="fa-brands fa-x-twitter"></i></a>
+          <a href="#"><i class="fa-solid fa-envelope"></i></a>
+          <a href="#"><i class="fa-regular fa-clipboard"></i></a>
+        </div>
+      </div>
+
+      <div class="imgs">
+        <img src="./img/img1.jpg" alt="" />
+        <div class="fingcap">
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release/A3-Press-Release.html:46]: Avoid to create classes just for one component, like imgs, fingcap, description, o about
Please Refactor it

Comment (Brandonz07) [reply src/br/html/A3-Press-Release/A3-Press-Release.html]: remove those classes

File: src/br/html/A3-Press-Release/A3-Press-Release.html [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+          <a href="#"><i class="fa-brands fa-x-twitter"></i></a>
+          <a href="#"><i class="fa-solid fa-envelope"></i></a>
+          <a href="#"><i class="fa-regular fa-clipboard"></i></a>
+        </div>
+      </div>
+
+      <div>
+        <img src="./img/img1.jpg" alt="" />
+        <div>
+          <figcaption class="img-gif txt-center">
```
Comment (luisalejandrosalazar) [inline src/br/html/A3-Press-Release/A3-Press-Release.html:47]: Avoid to create classes just for one component, like img-gif 
in this case, figcaption is unique in the document, apply properties to that tag
Please Refactor it

Comment (Brandonz07) [reply src/br/html/A3-Press-Release/A3-Press-Release.html]: remove class

**Review by luisalejandrosalazar — APPROVED** [review]
Approved.

**Pull request merged**